lib/components/app/batch-routing-panel.js

```diff
@@ -33,12 +33,13 @@ class BatchRoutingPanel extends Component {
       activeSearch,
       itineraryFooter,
       LegIcon,
+      ModeIcon,
       mobile,
       showUserSettings
     } = this.props
     const actionText = mobile ? 'tap' : 'click'
     return (
-      <ViewerContainer>
+      <ViewerContainer ModeIcon={ModeIcon}>
```

Comment:
Maybe out of scope for this PR, but is there a better way for us to pass around Leg/ModeIcon? Would it be useful to place them in the store on app startup (passed along with the OTP config object perhaps). I just find it a bit cumbersome to pass them to these connected components in this way.

Reply:
See #274 which proposes to address this comment.